examples/http_server/module/src/lib.rs

```diff
@@ -35,7 +35,8 @@ oak::entrypoint!(oak_main<ConfigMap> => |_receiver| {
 /// should be modified with care!
 pub struct StaticHttpServer;
 
-impl CommandHandler<Invocation> for StaticHttpServer {
+impl CommandHandler for StaticHttpServer {
+    type Command = Invocation;
```

Comment:
Perhaps add a blank line here to follow the same convention in the rest of the PR.

Reply:
Done.